OSPSuite.Presentation.Importer/Presenters/IColumnMappingPresenter.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using OSPSuite.Core.Importer;
+using OSPSuite.Presentation.Importer.Core;
+using OSPSuite.Presentation.Importer.Core.DataFormat;
+using OSPSuite.Presentation.Importer.Views;
+using OSPSuite.Presentation.Presenters;
+
+namespace OSPSuite.Presentation.Importer.Presenters
+{
+   /// <summary>
+   ///    Event arguments for OnMissingMapping event.
+   /// </summary>
+   public class MissingMappingEventArgs : EventArgs
+   {
+      /// <summary>
+      ///    FileName describing what is missed.
+      /// </summary>
+      public string Message { get; set; }
+   }
+
+   /// <summary>
+   ///    Handler for OnMissingMapping event.
+   /// </summary>
+   public delegate void MissingMappingHandler(object sender, MissingMappingEventArgs e);
+
+   public delegate void MappingCompletedHandler(object sender);
```

Comment:
use EVentHandler<>

Reply:
BTW this handler does nto coform to default EVentHandler signature

Reply:
done